client/config/config.go

```diff
@@ -20,6 +20,22 @@ var (
 		"AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN",
 		"GOOGLE_APPLICATION_CREDENTIALS",
 	}, ",")
+
+	// DefaulUserBlacklist is the default set of users that tasks are not
+	// allowed to run as when using a driver in "user.checked_drivers"
+	DefaultUserBlacklist = strings.Join([]string{
+		"root",
+		"Administrator",
+	}, ",")
+
+	// DefaultUserCheckedDrivers is the set of drivers we apply the user
+	// blacklist onto. For virtualized drivers it often doesn't make sense to
+	// make this stipulation so by default they are ignored.
+	DefaultUserCheckedDrivers = strings.Join([]string{
```

Comment:
Can we handle the situation when operators would want different lists of black-listed users for different driver?